docs/attack-discovery/attack-discovery.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[attack-discovery]]
+[chapter]
+= Attack Discovery
+
+:frontmatter-description: Accelerate threat identification by triaging alerts with a large language model.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [overview]
+:frontmatter-tags-user-goals: [get-started]
+
+beta::[]
+
+NOTE: This feature is available starting with {elastic-sec} version 8.14.0.
```

Comment:
We don't typically document the version in which a feature is introduced (it'd get unwieldy to do that for every feature). If there's a specific reason why this particular feature needs this, it's probably OK to keep the note, but otherwise I'd omit.

Reply:
Needs to be doc'd here for specific and idiosyncratic reasons. Will remove the note when possible

Reply:
🫡